### PR TITLE
Wire service & operation selectors to redux & url

### PR DESCRIPTION
**Pull request by everett980**
### Changes:
Service & Operation selectors now get their options from redux, and get their values from the url.

### CODE REVIEW

File: packages/jaeger-ui/src/components/DeepDependencies/Header/index.tsx [hunk 1]
```diff
@@ -13,70 +13,91 @@
...
+    }
+  }
 
-  setOperation = (operation: string) => {
-    // istanbul ignore next
-    this.setState({ operation });
+  setService = (service: string) => {
+    const { fetchServiceOperations, setService, operationsForService } = this.props;
+    if (!Reflect.has(operationsForService, service)) {
+      fetchServiceOperations(service);
```
Comment (tiffon) [inline packages/jaeger-ui/src/components/DeepDependencies/Header/index.tsx]: Having the `Header` fetch the services or operations, if they're needed, is inconsistent with `index.tsx` fetching the deps-graph data instead `Graph` taking care of that.

What do you think?

Comment (tiffon) [reply packages/jaeger-ui/src/components/DeepDependencies/Header/index.tsx]: Ping...

Comment (everett980) [reply packages/jaeger-ui/src/components/DeepDependencies/Header/index.tsx]: just pushed the updated version

**Comment by everett980** [comment]
Good point, it could possibly save on renders (e.g.: when changing service before `operationsForService` updates)
I'll update on the UiFind branch.

**Pull request merged**